src/App.tsx

```diff
@@ -15,47 +16,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  // eslint-disable-next-line no-console
+  console.log(sortType, isReversed);
```

Comment:
Remove the console.log statement as it is not necessary for production code and could lead to performance issues.